TaskManagement/templates/account/base.html

```diff
@@ -0,0 +1,73 @@
+
+
+{% load static%}
+{% load i18n %}
+<!doctype html>
+<html lang="en" dir="ltr">
+
+<!-- soccer/project/forgot-password.html  07 Jan 2020 03:42:43 GMT -->
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+<link rel="icon" href="favicon.ico" type="image/x-icon"/>
+
+<title>Logout</title>
```

Comment:
see how to use block this is not the correct way.
You can see the code of palak.